packages/pyroscope-flamegraph/src/format/format.ts

```diff
@@ -101,10 +101,12 @@ class DurationFormatter {
       nStr = '< 0.01';
     }
 
-    return `${nStr} ${
-      this.units ||
-      `${this.suffix}${n === 1 || this.suffix.length === 2 ? '' : 's'}`
-    }`;
+    return withUnits
+      ? `${nStr} ${
+          this.units ||
+          `${this.suffix}${n === 1 || this.suffix.length === 2 ? '' : 's'}`
+        }`
+      : nStr;
```

Comment:
https://github.com/pyroscope-io/pyroscope/pull/1565#discussion_r982492123

Reply:
i add `withUnits` so that `format` function can return formatted value but without units (seconds, minutes, hours, ..)